resources/views/profile/update-profile-information-form.blade.php

```diff
@@ -5,28 +5,66 @@
             <p class="card-description">{{ __('Update your account\'s profile information and email address.') }}</p>
         </div>
         <div class="card-body">
-            
+
             <x-maz-alert class="mr-3" on="saved" color='success'>
                 Saved
             </x-maz-alert>
             <form wire:submit.prevent="updateProfileInformation">
+
+                @if (Laravel\Jetstream\Jetstream::managesProfilePhotos())
+                <div x-data="{photoName: null, photoPreview: null}" class="col-6 col-sm-4">
+                    <input type="file" class="d-none" wire:model="photo" x-ref="photo" x-on:change="
+                                    photoName = $refs.photo.files[0].name;
+                                    const reader = new FileReader();
+                                    reader.onload = (e) => {
+                                        photoPreview = e.target.result;
+                                    };
+                                    reader.readAsDataURL($refs.photo.files[0]);
+                            " />
+
+                    <x-jet-label for="photo" value="{{ __('Photo') }}" />
+
+                    <div class="mt-2" x-show="! photoPreview">
+                        <img src="{{ $this->user->profile_photo_url }}" alt="{{ $this->user->name }}" width="250" height="250">
```

Comment:
I think it's better if you use avatar component that is provided by Mazer.

Reply:
Okay, Resolved